### PR TITLE
+ Added `disable_all_items` function in `discord.ui.View` and added `include_bots` parameter in `get_all_members`

### PR DESCRIPTION
**Pull request by 27Saumya**
# Warning: We have a feature freeze till release
That means we won't accept new features for now. Only bug fixes.

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

**Comment by krittick** [comment]
Can you please split this into 2 separate PRs since the two changes aren't related?

File: discord/client.py [hunk 1]
```diff
@@ -902,8 +908,15 @@ def get_all_members(self) -> Generator[Member, None, None]:
         :class:`.Member`
             A member the client can see.
         """
+        members: List[Member] = []
         for guild in self.guilds:
-            yield from guild.members
+            if include_bots == False:
```
Comment (Makiyu-py) [inline discord/client.py:913]:
```suggestion
            if not include_bots:
```

File: discord/client.py [hunk 2]
```diff
@@ -902,8 +908,15 @@ def get_all_members(self) -> Generator[Member, None, None]:
         :class:`.Member`
             A member the client can see.
         """
+        members: List[Member] = []
```
Comment (Makiyu-py) [inline discord/client.py:911]: why is this outside everything? This is only used in a single if-statement

**Comment by 27Saumya** [comment]
So shall i still make 2 different PRs or is it ok @krittick ?

**Comment by krittick** [comment]
> So shall i still make 2 different PRs or is it ok @krittick ?

Yes please. Since `get_all_members` has unresolved conversations, it's probably best to move `disable_all_items` to the second PR and change this one to only cover `get_all_members` (unless it's easier for you to remake the PR as two new ones).

**Pull request closed without merging**